recipes/denovogear/build.sh

```diff
@@ -0,0 +1,3 @@
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+cp -R ./* $outdir/
```

Comment:
You're right. Let me fix it. I didn't want to deal with the dependencies. But it will beneficial for sure.